test/e2e/shared/defaults.go

```diff
@@ -38,7 +38,7 @@ const (
 	OpenStackCloudYAMLFile     = "OPENSTACK_CLOUD_YAML_FILE"
 	OpenStackCloud             = "OPENSTACK_CLOUD"
 	OpenStackCloudAdmin        = "OPENSTACK_CLOUD_ADMIN"
-	OpenStackFailureDomain     = "OPENSTACK_FAILURE_DOMAIN"
+	OpenStackFailureDomain     = "OPENSTACK_FAILURE_DOMAIN" //nolint:gosec
```

Comment:
Ok it is `G101: Potential hardcoded credentials`. :facepalm: 
```suggestion
	OpenStackFailureDomain     = "OPENSTACK_FAILURE_DOMAIN" //nolint:gosec // Linter thinks this could be credentials...
```